docs/advanced/navigating-politeia-data.md

```diff
@@ -12,7 +12,7 @@ Below is a screenshot of the root directory of the [decred-proposals/mainnet](ht
 
 ![The root of the mainnet repository. * = proposal folders, A = anchoring data](/img/politeia/mainnet-pi-repo.png)
 
-Folders marked with `*` contain proposal data. Folders and files marked with `A` contain anchoring data. Anchoring data relates to the anchoring of Politeia data to the Decred blockchain through [dcrtime](https://github.com/decred/dcrtime), which facilitates Politeia's transparent censorship functionality. This guide will not describe those files in detail, as they are not likely to be of interest for analysis. 
+Folders marked with `*` contain proposal data. Folders and files marked with `A` contain anchoring data. Anchoring data relates to the anchoring of Politeia data to the Decred blockchain through [dcrtime](dcrtime.md), which facilitates Politeia's transparent censorship functionality. This guide will not describe those files in detail, as they are not likely to be of interest for analysis. 
```

Comment:
Can't see the png so make sure it matches the comments I made earlier.

Reply:
Both images in the dcrtime page were taken from the original [dcrtime blog](https://blog.decred.org/2017/06/14/Dcrtime/). I removed the Decred logo from both images, but otherwise kept them the same. Below are screenshots of both images.

![image](https://user-images.githubusercontent.com/1634777/60697434-1fae0e00-9e9f-11e9-925b-809b66641652.png)

![image](https://user-images.githubusercontent.com/1634777/60697443-29d00c80-9e9f-11e9-8102-c0d449be4c16.png)